amethyst_renderer/src/light.rs

```diff
@@ -173,3 +174,8 @@ impl From<SunLight> for Light {
         Light::Sun(sun)
     }
 }
+
+
+impl Component for Light {
+    type Storage = DenseVecStorage<Self>;
+}
```

Comment:
Probably want to fix these missing newlines.

Reply:
Fixed